src/zalsa_local.rs

```diff
@@ -158,18 +156,18 @@ impl ZalsaLocal {
     pub(crate) fn report_tracked_read(
         &self,
         input: DatabaseKeyIndex,
-        durability: Durability,
-        changed_at: Revision,
-        accumulated: InputAccumulatedValues,
+        revisions: &QueryRevisions,
         cycle_heads: &CycleHeads,
     ) {
         debug!(
             "report_tracked_read(input={:?}, durability={:?}, changed_at={:?})",
-            input, durability, changed_at
+            input, revisions.durability, revisions.changed_at
         );
+        // We don't access the accumulator anyways
+        let revisions = AssertUnwindSafe(revisions);
         self.with_query_stack(|stack| {
             if let Some(top_query) = stack.last_mut() {
-                top_query.add_read(input, durability, changed_at, accumulated, cycle_heads);
+                top_query.add_read(input, { revisions }.0, cycle_heads);
             }
         })
```

Comment:
I think I prefer the old method signature in that case to avoid this very *easy to get outdated* comment about accumulators. 

Not changing the signature (or only remuving accumulated) also has the benefit that it makes the diff a bit clearer.

Reply:
Went for that and `#[inline]`'d the function given ti has only one use site